### PR TITLE
Remove crypto usage from sigManager

### PR DESCRIPTION
**Pull request by yontyon**

### CODE REVIEW

File: util/include/crypto_utils.hpp [hunk 1]
```diff
@@ -20,19 +20,22 @@ enum class KeyFormat : std::uint16_t { HexaDecimalStrippedFormat, PemFormat };
 class IVerifier {
  public:
   virtual bool verify(const std::string& data, const std::string& sig) = 0;
+  virtual uint32_t signatureLength() = 0;
```
Comment (toly-kournik) [inline util/include/crypto_utils.hpp:23]: const

File: util/include/crypto_utils.hpp [hunk 2]
```diff
@@ -20,19 +20,22 @@ enum class KeyFormat : std::uint16_t { HexaDecimalStrippedFormat, PemFormat };
...
   virtual ~IVerifier() = default;
 };
 
 class ISigner {
  public:
   virtual std::string sign(const std::string& data) = 0;
+  virtual uint32_t signatureLength() = 0;
```
Comment (toly-kournik) [inline util/include/crypto_utils.hpp:30]: const

File: bftengine/src/bftengine/Crypto.hpp [hunk 3]
```diff
@@ -27,6 +27,7 @@ using std::string;
 
 namespace bftEngine {
 namespace impl {
+enum class KeyFormat : std::uint16_t { HexaDecimalStrippedFormat, PemFormat };
```
Comment (toly-kournik) [inline bftengine/src/bftengine/Crypto.hpp:30]: Do we still need it here?

Comment (yontyon) [reply bftengine/src/bftengine/Crypto.hpp:30]: just in the meantime, until we remove Crypto completely

**Review by toly-kournik — APPROVED** [review]
LGTM!

**Pull request merged**